cartography/intel/aws/dynamodb.py

```diff
@@ -10,32 +10,68 @@ def get_dynamodb_tables(session, region):
     paginator = client.get_paginator('list_tables')
     dynamodb_tables = []
     for page in paginator.paginate():
-        dynamodb_tables.extend(page['TableNames'])
-    return {'TableNames': dynamodb_tables}
+        for table in page:
+            dynamodb_tables.append(client.describe_table(table))
+    return {'Tables': dynamodb_tables}
 
 
 def load_dynamodb_tables(session, data, region, current_aws_account_id, aws_update_tag):
     ingest_table = """
     MERGE (table:DynamoDBTable{id: {Arn}})
     ON CREATE SET table.firstseen = timestamp(), table.arn = {Arn}, table.name = {TableName},
     table.region = {Region}
-    SET table.lastupdated = {aws_update_tag}
+    SET table.lastupdated = {aws_update_tag}, table.rows = {Rows}, table.size = {Size},
+    table.provisioned_throughput_read_capacity_units = {ProvisionedThroughputReadCapacityUnits},
+    table.provisioned_throughput_write_capacity_units = {ProvisionedThroughputWriteCapacityUnits}
     WITH table
     MATCH (owner:AWSAccount{id: {AWS_ACCOUNT_ID}})
     MERGE (owner)-[r:RESOURCE]->(table)
     ON CREATE SET r.firstseen = timestamp()
     SET r.lastupdated = {aws_update_tag}
     """
 
-    for table_name in data["TableNames"]:
-        arn = "arn:aws:dynamodb:{0}:{1}:table/{2}".format(region, current_aws_account_id, table_name)
+    for table in data["Tables"]:
         session.run(
             ingest_table,
-            Arn=arn,
-            TableName=table_name,
+            Arn=table['Table']['TableArn'],
+            TableName=table['Table']['TableName'],
             Region=region,
             AWS_ACCOUNT_ID=current_aws_account_id,
-            aws_update_tag=aws_update_tag
+            aws_update_tag=aws_update_tag,
+            Rows=table['Table']['ItemCount'],
+            Size=table['Table']['TableSizeBytes'],
+            ProvisionedThroughputReadCapacityUnits=table['Table']['ProvisionedThroughput']['ReadCapacityUnits'],
+            ProvisionedThroughputWriteCapacityUnits=table['Table']['ProvisionedThroughput']['WriteCapacityUnits']
+        )
+        load_gsi(session, region, current_aws_account_id, aws_update_tag, table)
+
+
+def load_gsi(session, region, current_aws_account_id, aws_update_tag, table):
+    ingest_gsi = """
+    MERGE (gsi:DynamoDBTableGSI{id: {Arn}})
```

Comment:
I think `DynamoDBIndex` is a clearer name for this node type. What do you think?

Since we're adding a new node type we should also add an index here: https://github.com/lyft/cartography/blob/master/cartography/data/indexes.cypher

Reply:
And the schema docs! https://github.com/lyft/cartography/blob/master/docs/schema/aws.md#dynamodbtable

Reply:
I found out that tables can contain two types of indexes, Local Secondary Indexes and Global Secondary Indexes. I think distinguishing the types might be useful if some team wanted to obtain information about a table's Local Secondary Indexes in the future. I'll make it `DynamoDBGlobalSecondaryIndex` for readability.